CUSTOM.md

```diff
@@ -0,0 +1,218 @@
+
+Aspects:
+- Add "Aspect" tag
+- Set name of object to that of aspect
+- Thresholds:
+  - Place on power card editor
+  - Follow power card threshold instructions
+- Add "Aspect" tag to deck of aspects as well if you have one of those
+
+Blight Cards:
+- Set card onto blight card editor
+- Change values that appear to match card
+- Set name of object to that of blight card
+- If blight card has recurring start of invader effect set object's description to that
+- You'll also want to make sure that Hide Face Down toggle is set for the card
+
+Expansions:
+- Create a bag
+- Name the bag the name of the expansion
+- Add the tag "Expansion" to the bag
+- Decks of cards need to be named one of the following:
+  - "Fear"
+  - "Events"
+  - "Blight Cards"
+  - "Minor Powers"
+  - "Major Powers"
+- Note using single cards in place of a deck may cause setup issues
+
+Adversary & Scenario:
+- Set name of object to that of adversary/scenario
+- collision
+  - workaround for TTS bug https://tabletopsimulator.nolt.io/770
+  - onObjectCollision: boolean - set to true if you have custom object collision code
+  - onObjectCollisionEnter(params)
+    - params: table - contains parameters for callback
+      - hit_object: object - object registered to receive collision events
+      - collision_info: table - table containing data about collision
+  - onObjectCollisionExit(params)
+    - params: table - contains parameters for callback
+      - hit_object: object - object registered to receive collision events
+      - collision_info: table - table containing data about collision
+- broadcast
+  - hasBroadcast: boolean - set to true if you have a Broadcast function
+  - Broadcast(params)
+      - params: table - contains data about adversaries (only exists for Adversary call)
+        - level: number - level of current adversary
+        - other: table - contains data about other adversary
+          - exist: boolean - whether there is a second adversary
+          - level: number - other adversary level
+    - return string - what players should be told, nil otherwise
+- Callbacks:
+  - requirements - determine whether random adversary/scenario should be used with current setup
+    - requirements: boolean - set to true if you have Requirements function
+    - Requirements(params)
+      - params: table - contains data about game setup
+        - eventDeck: boolean - whether event deck is being used
+        - blightCard: boolean - whether blight card is being used
+        - expansions: table - expansions included in setup, keys are the name of them
+        - thematic: boolean - whether thematic island setup is enabled
+        - adversary: boolean - whether an adversary is being used (only exists for Scenario call)
+      - return boolean - whether this adversary/scenario should be used
+  - preSetup - called before all other setup steps have started
```

Comment:
Needs an example of when it would be needed. I think it's for invader piece adjustment, right?

Reply:
added a blurb in there for swapping out bags, powers long forgotten also uses this to prepare the scenario token bag iirc